### ISSUE TITLE
compileModules: true doesn't work

### ISSUE THREAD

**Issue by mitchellmebane** [babel/ember-cli-babel#18]
I spun up a new `ember-cli` app, installed `ember-cli-babel`, and added `'babel': { compileModules: true }` in my `Brocfile.js`.  When I run `ember server`, I get:

```
TypeError: Object #<Object> has no method 'indexOf'
    at getOptions (/var/node/testapp/node_modules/ember-cli-babel/index.js:28:40)
    at Class.module.exports.included (/var/node/testapp/node_modules/ember-cli-babel/index.js:6:19)
    at EmberApp.<anonymous> (/var/node/testapp/node_modules/ember-cli/lib/broccoli/ember-app.js:277:15)
    at Array.filter (native)
    at EmberApp._notifyAddonIncluded (/var/node/testapp/node_modules/ember-cli/lib/broccoli/ember-app.js:272:45)
    at new EmberApp (/var/node/testapp/node_modules/ember-cli/lib/broccoli/ember-app.js:236:8)
    at Object.<anonymous> (/var/node/testapp/Brocfile.js:5:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

It looks to me like this line in index.js is the culprit:

    options.blacklist.splice(options.indexOf('es6.modules'), 1);

I think it should be:

    options.blacklist.splice(options.blacklist.indexOf('es6.modules'), 1);

---

Something is still missing, though, because changing that leads to more errors:

```
ReferenceError: Unknown option: compileModules
    at File.normalizeOptions
```

**Comment by mitchellmebane** [comment]
I added `delete options.compileModules`, and I *think* it's working properly now.  I guess I need to actually proceed with learning Ember now...

```javascript
var blacklistOptionIdx = options.blacklist.indexOf('es6.modules');
if (options.compileModules === true) {
  if (blacklistOptionIdx >= 0) {
    options.blacklist.splice(blacklistOptionIdx, 1);
  }
  delete options.compileModules;
} else {
  if (blacklistOptionIdx < 0) {
    options.blacklist.push('es6.modules');
  }
}
```

**Comment by stefanpenner** [comment]
why not let ember-cli continue to handle the module transpilation?

**Comment by mitchellmebane** [comment]
No real reason.

**Comment by stefanpenner** [comment]
our plan is to continue to utilize esperanto which will allow folding and other interesting bundled outputs. Currently babel does not aim to support this.

**Comment by gordonkristan** [comment]
Closed by #20

**Comment by mitchellmebane** [comment]
Well, if there's no benefit to doing modules with Babel, then I won't bother.  But thanks for fixing the issue until such time as the feature (or at least the documentation for it) is removed, as it does make things less confusing for a new user like me who is stumbling around and assuming that anything that blows up is my fault.
